compute_tools/src/http/openapi_spec.yaml

```diff
@@ -68,6 +68,44 @@ paths:
               schema:
                 $ref: "#/components/schemas/Info"
 
+  /schema/objects:
+    get:
+      tags:
+        - Info
+      summary: Get databases and roles in the schema.
+      description: ""
+      operationId: getSchemaObjects
+      responses:
+        200:
+          description: Compute schema objects
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/SchemaObjects"
+
+  /schema/dump:
```

Comment:
I think `/schema/sql` or `/schema/ddl`  might be a better endpoint name.

Reply:
done